### ISSUE TITLE
date('n') output format gives month number instead of English three-character month abbreviation

### ISSUE THREAD

**Issue by rvjansen** [adesutherland/CREXX#228]
as in subject.

**Comment by Peter-Jacob** [comment]
error confirmed. corrected in: commit 1f75246d11b356e94987082ad5197ab6f2b8802f
Please verify and close issue.

examples:
```
say "date()" date()
say "date('USA')" date('USA')
say "date('normal')" date('n')
say "date('qualified')" date('QUALIFIED')
say "date('European')" date('EUROPEAN')
say "date('Epoch')" date('EPOCH')
say "date('xGerman')" date('XGERMAN')
say "date('Julian')" date('JULIAN')
say "date('Ordered')" date('ORDERED')
say "date('International')" date('INTERNATIONAL')
say "date('Sorted')" date('SORTED')
```
Result:
```
date() 26 Nov 2021
date('USA') 11/26/2021
date('normal') 26 Nov 2021
date('qualified') Friday, November 26, 2021
date('European') 26/11/21
date('Epoch') 1637884800
date('xGerman') 26.11.2021
date('Julian') 2021330
date('Ordered') 21/11/26
date('International') 2021-11-26
date('Sorted') 20211126
```

**Comment by rvjansen** [comment]
works as designed
